server/channels/app/channel.go

```diff
@@ -3723,13 +3800,15 @@ func (a *App) validateForConvertGroupMessageToChannel(c request.CTX, convertedBy
 		)
 	}
 
-	channelMember, appErr := a.GetChannelMember(c, gmConversionRequest.ChannelID, convertedByUserId)
-	if appErr != nil {
-		return appErr
-	}
+	if convertedByUserId != "" {
+		channelMember, appErr := a.GetChannelMember(c, gmConversionRequest.ChannelID, convertedByUserId)
```

Comment:
What was the purpose of this check originally? To check whether the user making the request to convert the channel is a member of the channel or not? 

cc @harshilsharma63